pkg/activator/net/throttler.go

```diff
@@ -251,24 +251,75 @@ func (rt *revisionThrottler) updateThrottleState(throttler *Throttler, backendCo
 	}
 }
 
+// pickIndices picks the indices for the slicing.
+func pickIndices(lt, selfIndex, numActivators int) (bi, ei int) {
+	if numActivators > lt {
+		// 1. We have fewer pods than than activators. Assign the pod in round robin fashion.
+		// NB: when we implement subsetting this will be less of a problem.
+		// e.g. lt=3, #ac = 5; for si = 3 => 3 % 3 = 0, or for si = 5 => 5%3 = 2
```

Comment:
What is `si`

Reply:
selfIndex (shorthand for the parameters)